pilot/pkg/util/informermetric/informerutil.go

```diff
@@ -0,0 +1,61 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package informermetric
+
+import (
+	"sync"
+
+	"k8s.io/client-go/tools/cache"
+
+	"istio.io/pkg/log"
+	"istio.io/pkg/monitoring"
+)
+
+var (
+	clusterLabel = monitoring.MustCreateLabel("cluster")
+
+	errorMetric = monitoring.NewSum(
+		"controller_sync_errors",
```

Comment:
```suggestion
		"controller_sync_errors_total",
```
per https://prometheus.io/docs/practices/naming/#metric-names